lib/ohai/plugins/linux/platform.rb

```diff
@@ -59,6 +59,15 @@ def get_redhatish_version(contents)
       contents = File.read("/etc/parallels-release").chomp
       platform get_redhatish_platform(contents)
       platform_version contents.match(/(\d\.\d\.\d)/)[0]
+    elsif File.exists?('/etc/os-release')
```

Comment:
It needs to go before the Red Hat detection because Wind River has a /etc/redhat file despite not being a Red Hat-based distro. I could make it smarter and put the logic inside the Red Hat detection.